### ISSUE TITLE
Squiz.Commenting.VariableComment not checking var types when the @var line contains a comment

### ISSUE THREAD

**Issue by josephzidell** [squizlabs/PHP_CodeSniffer#2115]
Code
------
```php
/**
 * @var int Type
 */
public $type;
```

Actual Result
--------------
Produces no error

Expected Result
-----------------
Expected "integer" but found "int" for @var tag in member variable comment

**Comment by josephzidell** [comment]
Turns out the case covered in `src\Standards\Squiz\Tests\Commenting\VariableCommentUnitTest.php` for line 136 is for a multiline comment.

Will create a PR with test case.

**Comment by gsherwood** [comment]
This should now be fixed. The linked PR fixed this specific case but not more complex type cases. I ported over code from the FunctionComment sniff to resolve the issue. Thanks for the report.

**Comment by josephzidell** [comment]
Thanks @gsherwood !
